packages/edit-post/src/hooks/commands/use-common-commands.js

```diff
@@ -194,16 +194,16 @@ export default function useCommonCommands() {
 	useCommand( {
 		name: 'core/toggle-publish-sidebar',
 		label: isPublishSidebarEnabled
-			? __( 'Disable pre-publish checklist' )
-			: __( 'Enable pre-publish checklist' ),
+			? __( 'Disable pre-publish checks' )
+			: __( 'Enable pre-publish checks' ),
 		icon: formatListBullets,
 		callback: ( { close } ) => {
 			close();
 			toggle( 'core/edit-post', 'isPublishSidebarEnabled' );
 			createInfoNotice(
 				isPublishSidebarEnabled
-					? __( 'Pre-publish checklist off.' )
-					: __( 'Pre-publish checklist on.' ),
+					? __( 'Pre-publish checks off.' )
```

Comment:
How can I trigger this info notice? It seems to only be used with the command palette? Perhaps this could be more consistent, but it is not a blocker for the PR.

As a non-native English speaker, this text, especially "checks off" reads a bit confusing to me.
"Pre-publish checklist is disabled"  and "Pre-publish checklist is enabled" would be easier for me to read.

Reply:
Yes they are triggered only when using the command palette.
I'm not sure they need to be used also when using the preference menu. That's a native `input type="checkbox"` which is natively accessible.
Good point on improving the text, `are disabled` sounds better. Thanks for the review!